about.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      href="http://fonts.googleapis.com/css?family=Lato:400,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/brands.min.css"
+      integrity="sha512-+oRH6u1nDGSm3hH8poU85YFIVTdSnS2f+texdPGrURaJh8hzmhMiZrQth6l56P4ZQmxeZzd2DqVEMqQoJ8J89A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+      href="http://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/ce6302c6e5.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>About</title>
+  </head>
+  <body>
+     <header class="header"> 
+      <ul>
+        <li><a href="#" class="fa fa-twitter"></a> </li>
+        <li><a href="#" class="fa fa-facebook"></a> </li>
+
+        <li>English</li>
+        <li>My page</li>
+        <li>Layout</li>
+        
+      </ul>
+    </header>
+    <nav class="navbar-Mobile">
+      <div class="logonav">
+    <a href="about.html">
+        <img src="images/logo.avif" class="logonav" alt=""></a>
+      </div>
+     <ul class="menu">
+       <li><a href=""><img class="closebtn" src="images/AppBar.png" alt=""></a></li>
+       <li><a href="index.html">About</a></li>
+       <li><a href="">program</a></li>
+       <li><a href="">News</a></li>
+       <li><a href="">Join</a></li>
+       <li><a href="">Sponsor</a></li>
+       <li><a href="">Compaign</a></li>
+     </ul>
+          <div class="hamburger">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+        </div>
+    </nav>
+    <div class="wrapper">
+        
+      <div class="intro">
+        
+        <h3 class="title-about">"Hello Freshers!"</h3>
+        <h1 class="about-h1">Mobile first design strategy</h1>
+        <p class="about-info">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Impedit
+          magnam odit praesentium excepturi eveniet, dolor dolorem ex incidunt
+          perferendis
+        </p>
+        <p class="caption-about">2020/30/10 Join us for better knowledge</p>
+        <p class="caption-about1">@Kabul University Afghanistan</p>
+      </div>
```

Comment:
- [ ] `required`
- [ ] Good so far 👏
- [ ] however please see to it that Layout:
- [ ]  composition and space between elements are the same.
- [ ] The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
- [ ] Please use the `social icons` as the template provided which doesn't have lines at the bottom.
- [ ] Kindly increase the `fonts` and the `spacing` to fit the template
- [ ] Please see screenshots below
`your template`
![1st page](https://user-images.githubusercontent.com/66230306/190277102-ecd8a2f8-81cd-4289-87cd-65ee55fb8f13.png)

`Provided template`
![image](https://user-images.githubusercontent.com/66230306/190278184-984aa57d-bf10-48a9-9d98-f8288a1d6522.png)